FEXCore/Source/Interface/HLE/Thunks/Thunks.cpp

```diff
@@ -274,135 +269,130 @@ namespace FEXCore {
-                    That->Thunks[*reinterpret_cast<IR::SHA256Sum*>(Exports[i].sha256)] = Exports[i].Fn;
-                }
+      int i;
+      for (i = 0; Exports[i].sha256; i++) {
+        That->Thunks[*reinterpret_cast<IR::SHA256Sum*>(Exports[i].sha256)] = Exports[i].Fn;
+      }
 
-                LogMan::Msg::DFmt("Loaded {} syms", i);
-            }
-        }
+      LogMan::Msg::DFmt("Loaded {} syms", i);
+    }
+  }
 
-        static void IsLibLoaded(void* ArgsRV) {
-            struct ArgsRV_t {
-                const char *Name;
-                bool rv;
-            };
+  static void IsLibLoaded(void* ArgsRV) {
+    struct ArgsRV_t {
+      const char* Name;
+      bool rv;
+    };
 
-            auto &[Name, rv] = *reinterpret_cast<ArgsRV_t*>(ArgsRV);
+    auto& [Name, rv] = *reinterpret_cast<ArgsRV_t*>(ArgsRV);
 
-            auto CTX = static_cast<Context::ContextImpl*>(Thread->CTX);
-            auto That = reinterpret_cast<ThunkHandler_impl*>(CTX->ThunkHandler.get());
+    auto CTX = static_cast<Context::ContextImpl*>(Thread->CTX);
+    auto That = reinterpret_cast<ThunkHandler_impl*>(CTX->ThunkHandler.get());
 
-            {
-                std::shared_lock lk(That->ThunksMutex);
-                rv = That->Libs.contains(Name);
-            }
-        }
+    {
+      std::shared_lock lk(That->ThunksMutex);
+      rv = That->Libs.contains(Name);
+    }
+  }
 
-        ThunkedFunction* LookupThunk(const IR::SHA256Sum &sha256) override {
+  ThunkedFunction* LookupThunk(const IR::SHA256Sum& sha256) override {
 
-            std::shared_lock lk(ThunksMutex);
+    std::shared_lock lk(ThunksMutex);
 
-            auto it = Thunks.find(sha256);
+    auto it = Thunks.find(sha256);
 
-            if (it != Thunks.end()) {
-                return it->second;
-            } else {
-                return nullptr;
-            }
-        }
+    if (it != Thunks.end()) {
+      return it->second;
+    } else {
+      return nullptr;
+    }
+  }
 
-        void RegisterTLSState(FEXCore::Core::InternalThreadState *_Thread) override {
-          Thread = _Thread;
-        }
+  void RegisterTLSState(FEXCore::Core::InternalThreadState* _Thread) override { Thread = _Thread; }
 
-        void AppendThunkDefinitions(fextl::vector<FEXCore::IR::ThunkDefinition> const& Definitions) override {
-          for (auto & Definition : Definitions) {
-            Thunks.emplace(Definition.Sum, Definition.ThunkFunction);
-          }
-        }
-    };
-
-    fextl::unique_ptr<ThunkHandler> ThunkHandler::Create() {
-      return fextl::make_unique<ThunkHandler_impl>();
+  void AppendThunkDefinitions(fextl::vector<FEXCore::IR::ThunkDefinition> const& Definitions) override {
+    for (auto& Definition : Definitions) {
+      Thunks.emplace(Definition.Sum, Definition.ThunkFunction);
     }
+  }
+};
+
+fextl::unique_ptr<ThunkHandler> ThunkHandler::Create() { return fextl::make_unique<ThunkHandler_impl>(); }
 
-    /**
+/**
```

Comment:
clang-format un-indented only the first line of this docstring. Looks like we'll need to move the other lines manually :(

Reply:
Solved as well.